README.md

```diff
@@ -1,24 +1,52 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Budget App
 
-Things you may want to cover:
+The Ruby on Rails capstone project is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-* Ruby version
+## Getting Started
+To get a local copy up and running follow these simple example steps.
 
-* System dependencies
+- Copy this link `https://github.com/Ghilain/budget-app.git`
+- Get the directory that you want to clone the repository.
+- Open the command prompt in this directory.
+- Write `https://github.com/Ghilain/budget-app.git`.
+- Go to the repository folder in your command prompt `cd budget-app`.
 
```

Comment:
- [ ] Please update your Readme, currently, you're missing the `Built With` section. Kindly add this and make sure to list all technologies and languages you used to implement this app ✍🏻